src/reactviews/pages/QueryResult/commandBar.tsx

```diff
@@ -64,13 +64,28 @@ const CommandBar = (props: CommandBarProps) => {
         });
     };
 
-    const hasMultipleResults =
-        context.state.resultSetSummaries &&
-        Object.keys(context.state.resultSetSummaries).length > 1;
+    const checkMultipleResults = () => {
+        if (Object.keys(context.state.resultSetSummaries).length > 1) {
+            return true;
+        }
+        for (let resultSet of Object.values(context.state.resultSetSummaries)) {
+            if (Object.keys(resultSet).length > 1) {
+                return true;
+            }
+        }
+        return false;
+    };
+
+    const hasMultipleResults = () => {
+        return (
+            Object.keys(context.state.resultSetSummaries).length > 0 &&
```

Comment:
The key results to `true`? Is it possible to be in a situation where the key is reused and would that cause values to be overwritten?